src/Microsoft.ML.Transforms/ProduceIdTransform.cs

```diff
@@ -25,10 +25,15 @@ namespace Microsoft.ML.Runtime.Data
     /// </summary>
     public sealed class ProduceIdTransform : RowToRowTransformBase
     {
+        private static class Defaults
+        {
+            public const string Column = "Id";
+        }
```

Comment:
I think we probably don't need this any longer.

Reply:
Indeed we might as well revert the whole file probably.

---
In reply to: [202426903](https://github.com/dotnet/machinelearning/pull/520#discussion_r202426903) [](ancestors = 202426903)